recipes/rust/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.18.0" %}
+
+package:
+  name: rust
+  version: {{ version }}
+
+source:
+  fn: rust-{{ version }}.tar.gz
+  url: https://static.rust-lang.org/dist/rustc-{{ version }}-src.tar.gz
+  sha256: d2dc36e99b9e2269488b2bcddde43c234e6bde03edf70cba82a027ff49c36111
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - llvmdev 3.9.*
+    - toolchain
+    - python 2.7*
+    - cmake
+    - openssl
+    - curl
+    - pkg-config  # [osx]
+  run:
+    - openssl
```

Comment:
Looks like you need `curl` and `zlib`. Both of them and `openssl` need to be pinned in `run` and `build`